test_scripts/PROJECTION/000_Start_secured_audio_service.lua

```diff
@@ -0,0 +1,63 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1) Application is registered with PROJECTION appHMIType
+-- 2) and starts secured audio service
+-- SDL must:
+-- 1) Start services successful
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/PROJECTION/common')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local appHMIType = "PROJECTION"
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appName = "server"
+config.application1.registerAppInterfaceParams.appID = "SPT"
+config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
+config.defaultProtocolVersion = 3
+
+--[[ Local Functions ]]
+local function ptUpdate(pTbl)
+  local filePath = "./files/Security/client_credential.pem"
+  local crt = common.readFile(filePath)
+  pTbl.policy_table.module_config.certificate = crt
+  pTbl.policy_table.app_policies[common.getAppID()].AppHMIType = { appHMIType }
+end
+
+local function startServiceSecured(pData, serviceId)
+  common.getMobileSession():StartSecureService(serviceId)
+  common.getMobileSession():ExpectControlMessage(serviceId, pData)
+  local handshakeOccurences = 0
+  if pData.encryption == true then handshakeOccurences = 1 end
+  common.getMobileSession():ExpectHandshakeMessage()
+  :Times(handshakeOccurences)
+  common.delayedExp()
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Set ForceProtectedService OFF", common.setForceProtectedServiceParam, { "Non" })
```

Comment:
@GetmanetsIrina Setting the parameter to "Non" is not required since this is default value

Reply:
locally changed.